editions/tw5.com/tiddlers/filters/examples/substitute Operator (Examples).tid

```diff
@@ -0,0 +1,37 @@
+created: 20230614225302905
+modified: 20230614233448662
+tags: [[Operator Examples]] [[substitute Operator]]
+title: substitute Operator (Examples)
+type: text/vnd.tiddlywiki
+
+\define time() morning
+\define field() modified
+\define sentence() This tiddler was last $(field)$ on ${[{!!modified}format:date[DDth MMM YYYY]]}$
```

Comment:
```suggestion
\procedure sentence() This tiddler was last $(field)$ on ${[{!!modified}format:date[DDth MMM YYYY]]}$
```

Without the above, I'm seeing the following in the display:
```
This tiddler was last modified on ${[{!!modified}format:date[DDth MMM YYYY]]}$
```

Reply:
thanks @btheado